spock-core/core.gradle

```diff
@@ -13,6 +13,7 @@ Spock is inspired from JUnit, jMock, RSpec, Groovy, Scala, Vulcans, and other fa
 dependencies {
   compile libs.groovy // easiest way to add Groovy dependency to POM
   compile libs.junit
+  compile libs.annotations
```

Comment:
Just look two lines below `compile libs.annotations, optional` and I think it should be called `intellijAnnotations`. 
> Not sure what you mean by forcing it on the user 
I mean we increase the amount of dependencies, even for users who use, e.g., eclipse and don't profit from it.